src/components/Wizard/CreateVmWizard/providers/tests/vmwareProviderPod.test.js

```diff
@@ -0,0 +1,63 @@
+import { getDefaultSecretName, getImportProviderSecretObject, getV2VVMwareObject } from '../vmwareProviderPod';
+import { SecretModel, V2VVMwareModel } from '../../../../../models';
+import { VCENTER_TEMPORARY_LABEL } from '../../../../../constants';
+
+describe('VMWareProviderPod()', () => {
+  it('returns default secret name correctly', () => {
+    expect(getDefaultSecretName({ username: 'a', url: 'url' })).toBe('url-a');
+    expect(getDefaultSecretName({ username: 'a', url: 'https://url' })).toBe('url-a');
+    expect(getDefaultSecretName({ username: 'a', url: 'http://url' })).toBe('url-a');
+    expect(getDefaultSecretName({ username: 'a', url: 'https://url.com/' })).toBe('url.com-a');
+    expect(getDefaultSecretName({ username: 'a', url: 'https://url.com/foo' })).toBe('url.com-a');
+    expect(getDefaultSecretName({ username: 'a@b', url: 'https://url.com/foo/bar' })).toBe('url.com-a-at-b');
```

Comment:
would be good to check the exceptions here as well

Reply:
good point, adding the test